### ISSUE TITLE
val loss did not go down

### ISSUE THREAD

**Issue by ruoyangliu** [ruoyangliu/cross#1]
if I train the newtork with pre-generated label for val the loss for val is not go down
